test/specs/calculator.add.multiply.spec.js

```diff
@@ -0,0 +1,158 @@
+    // eslint-disable-next-line max-len
+    expect(multiplier.multiply(-245.969, 138.354)).to.be.equal(-34030.795026);
+  });
+});
+
+// eslint-disable-next-line max-len
+describe('It should test Calculator class method \'multiply\' negative scenarios', () => {
+  let multiplier;
+  beforeEach(() => {
+    multiplier = new Calculator();
+  });
+  afterEach(() => {
+    multiplier = null;
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' is not defined(null) an error is thrown', () => {
+    expect(() => multiplier.multiply(68, null)()).to.throw('Unable to multiply, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'increment\' is not defined(null) an error is thrown', () => {
+    expect(() => multiplier.multiply(68, null)()).to.throw('Unable to multiply, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' and \'multiplier\' are not defined(null) an error is thrown', () => {
+    expect(() => multiplier.multiply(null, null)()).to.throw('Unable to multiply, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' and \'multiplier\' are not defined(values are empty) an error is thrown', () => {
+    expect(() => multiplier.multiply()()).to.throw('Unable to multiply, number and multiplier are not defined');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if the only \'number\' is defined an error is thrown', () => {
+    expect(() => multiplier.multiply(5)()).to.throw('Unable to multiply, only one number is defined');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if the only \'number\' is defined(null) an error is thrown', () => {
+    expect(() => multiplier.multiply(null)()).to.throw('Unable to multiply, [null] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if the only \'number\' is defined(as undefined) an error is thrown', () => {
+    expect(() => multiplier.multiply(undefined)()).to.throw('Unable to multiply, [undefined] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' and \'multiplier\' are not defined(undefined) an error is thrown', () => {
+    expect(() => multiplier.multiply(undefined, undefined)()).to.throw('Unable to multiply, [undefined] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'number\' is not defined(undefined) an error is thrown', () => {
+    expect(() => multiplier.multiply(undefined, 5)()).to.throw('Unable to multiply, [undefined] is not a number');
+  });
+  // eslint-disable-next-line max-len
+  it('It should test that if \'increment\' is not defined(undefined) an error is thrown', () => {
+    expect(() => multiplier.multiply(68, undefined)()).to.throw('Unable to multiply, [undefined] is not a number');
+  });
```

Comment:
Please add negative tests with other types of arguments: `string`, `object`, etc.